### DISCUSSION TITLE
새벽 동안에 내부서버가 내려가 있었나요?

### DISCUSSION THREAD

**Discussion by Cha-SeongHun** [naverz/zepeto-studio-kor#1264] [5. Q&A]
월드를 개발 중 동일한 빌드 파일로 여러 번 테스트를 해보았습니다.
9월 6일 새벽 (4시 ~ 9시) 동안 아래 증상과 함께 동작하지 않던 월드가 있습니다.
- 접속 시 유저목록에 아무도 나타나지 않고 자신 또한 표시되지 않음
- 대화 UI 버튼이 눌리지 않음
- 모든 스크립트가 동작하지 않음
- 내부 C#스크립트 QualitySettingExtension 의 생성자에서 예외가 발생함

 그러나 동일한 월드가 지금은 정상적으로 동작이 되고 있습니다.
정확한 원인 트래킹을 목적으로 새벽동안 서버가 내려가 있었던 것인지 여쭤봅니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 해당 시간에 서버가 내려가 있지는 않았습니다.🥲
말씀하신 문제의 원인은 복합적이라 현상만으로는 파악이 어려울 것 같습니다.
추후에 같은 현상이 발생하게 되었을때 내부 상황도 함께 공유해주시면 좋을 것 같습니다.
감사합니다.

**Reply by Cha-SeongHun** [reply]
확인해 주셔서 감사합니다!
